src/index.js

```diff
@@ -4,62 +4,84 @@ import NavBar from './nav-bar.js'
 import ViewCards from './view-cards.js'
 import CardForm from './card-form.js'
 import PracticeCards from './practice-cards.js'
+import uuid from 'uuid/v4'
```

Comment:
Typically we allow the server to determine the id of a resource. With UUIDs, it's ok to do it client-side, because they are "universally unique", but often they're auto-incrementing integers and that can't be reliably set by the client.